w03-intro-backend-with-express/d4-dusk-mongoose/exercises.md

```diff
@@ -0,0 +1,21 @@
+# Express Mongoose Exercises
+
+### Base Challenges
+
+**Adding Mongoose to toEatly**
+
+1. Clone [this repo](https://github.com/sf-wdi-22-23/toEatly_mongoose/tree/master) as starter code. Run `npm install`.
+1. Install `nodemon` globally on your computer
+1. Install `mongo` - follow [these instructions](http://docs.mongodb.org/manual/tutorial/install-mongodb-on-os-x/)
+1. Run mongo on your computer with the `mongod` method.
+1. Install `mongoose` locally to your project with npm. require and connect to your `mongoose` database.
+1. Create a `Food` Model & Schema
+1. Seed your database with `seed.js`
+1. Use your new `Food` model in your Index route to send your foods to the client.
+1. Update your Create route to use your new `Food` model to save new food document to your food collection to your DB.
```

Comment:
Based on the one below, text is probably meant to say "save *a* new food document to your food collection *in* your DB."